package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@promotedai/react-native-metrics",
-  "version": "1.2.4",
+  "version": "1.3.0",
```

Comment:
This was done in main a long time ago. Curious how you're seeing this old version in the repo.

https://github.com/promotedai/react-native-metrics/commit/fde69a8be2adfadbc046759bbb6650ad87b84a64

Reply:
Weird.  Not sure.